src/components/Category/CategoryBanner.tsx

```diff
@@ -8,26 +8,35 @@ import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
 import './CategoryBanner.css'
 
 const icons = {
-  [ProposalType.Catalyst]: require('../../images/icons/catalyst.svg'),
-  [ProposalType.POI]: require('../../images/icons/poi.svg'),
-  [ProposalType.BanName]: require('../../images/icons/ban-name.svg'),
-  [ProposalType.Grant]: require('../../images/icons/grant.svg'),
-  [ProposalType.Poll]: require('../../images/icons/poll.svg'),
+  [ProposalType.Catalyst]: require("../../images/icons/catalyst.svg"),
+  [ProposalType.POI]: require("../../images/icons/poi.svg"),
+  [ProposalType.AddPOI]: require("../../images/icons/add-poi.svg"),
+  [ProposalType.RemovePOI]: require("../../images/icons/remove-poi.svg"),
+  [ProposalType.BanName]: require("../../images/icons/ban-name.svg"),
+  [ProposalType.Grant]: require("../../images/icons/grant.svg"),
+  [ProposalType.Poll]: require("../../images/icons/poll.svg"),
 }
 
-export type CategoryBannerProps = Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, 'children'> & {
+export type CategoryBannerProps = Omit<
```

Comment:
Please disable the auto-formatting so it's easier for reviewers to tell which lines are edited 😄